src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Rectangle class should not extend Figure if Figure is an abstract class meant to define behavior, as per the checklist. Consider using an interface or making Figure a concrete class instead.